src/screens/ProfileScreen/index.js

```diff
@@ -709,7 +709,7 @@ const ProfileScreen = ({route}) => {
         onClose={onHeaderOptionClosed}
         onDeleteClicked={onDeletePost}
       />
-    </SafeAreaProvider>
+    </View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The closing tag for `SafeAreaProvider` has been replaced with a closing tag for `View`. Ensure that this change does not affect the layout on devices with notches or rounded corners.
```diff
- </SafeAreaProvider>
+ </View>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->